<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>asignar_materias</class>
 <widget class="QDialog" name="asignar_materias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f6fa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetPrincipal" native="true">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: #F7F9FC;
	font-family: &quot;Segoe UI&quot;;
color: #2d2d2d;
}
QLineEdit {
    border: 2px solid #2980b9;
    border-radius: 10px;
    padding: 2px 8px;
    background-color: white;
	color: #2d2d2d;
}
QPushButton {
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 10px;
    border-radius: 12px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="lblTitulo_asignar_materia">
        <property name="minimumSize">
         <size>
          <width>241</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Asignar materias</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QScrollArea" name="scrollArea_asignar_materia">
        <property name="minimumSize">
         <size>
          <width>361</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>361</width>
          <height>330</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>359</width>
           <height>328</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_asignar_materia" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="chkAsignar_materia">
               <property name="text">
                <string>CheckBox</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSelec_todas">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    spacing: 8px;
    color: #2d2d2d;
}

QCheckBox::indicator {
    width: 22px;
    height: 22px;
    border: 2px solid #2c3e50;
    border-radius: 6px;
    background: #ffffff;
}

QCheckBox::indicator:hover {
    border: 2px solid #1565C0;
}

QCheckBox::indicator:checked {
    border: 2px solid #1565C0;
    background: #2980b9;  /* mantener blanco para que Qt pinte la palomita */
}</string>
        </property>
        <property name="text">
         <string>Seleccionar todas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>361</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>361</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnGuardar_asignar_materias">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>120</width>
           <height>35</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
         <property name="icon">
          <iconset resource="../icons/resources_ui.qrc">
           <normaloff>:/icons/confirm_white.png</normaloff>:/icons/confirm_white.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="btnCancelar_asignar_materias">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>120</width>
           <height>35</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: white;
color: #2980b9;
border: 1.5px solid #2980b9;</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
         <property name="icon">
          <iconset resource="../icons/resources_ui.qrc">
           <normaloff>:/icons/cancelar_b2.png</normaloff>:/icons/cancelar_b2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resources_ui.qrc"/>
 </resources>
 <connections/>
</ui>
